test/media-card.test.js

```diff
@@ -0,0 +1,38 @@
+/* global expect */
+/* eslint-disable func-names, prefer-arrow-callback */
+/* eslint import/no-extraneous-dependencies: ["error", { "devDependencies": true  }] */
+import { shallow } from 'enzyme';
+import React from 'react';
+import sinon from 'sinon';
+import MediaCard from '../src/components/media-card';
+
+const imageSrc = 'https://www.test.png';
+const videoSrc = 'https://www.test.mp4';
```

Comment:
Could you make these just `'test.png'` and `'test.mp4'`? The mix of the url and file extension is a bit odd and potentially confusing (The file extensions are in the place of what would be the top level domain in a URL).

Reply:
Makes sense.